<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3DViewer_v1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_File">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Screenshot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>690</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_gif">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>690</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gif</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_vert">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>620</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Num of Vertices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_faces">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>650</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Num of painted Edges:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trans">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>565</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Transition</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>620</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>650</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>650</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_turn">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>565</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>620</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>570</y>
      <width>240</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_val_num_faces">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_val_num_vert">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_info">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>-4</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_val_name">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>570</y>
      <width>91</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_trans_x">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>50</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_trans_y">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>50</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_trans_z">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>50</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>90</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_turn_y">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>50</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_turn_z">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>50</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>570</y>
      <width>140</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox_edge">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dotted</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_edge_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SpinBox_edge_size">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_edges">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>565</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Edges</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>590</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertices">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>565</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertices</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>140</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox_vertices">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Circle</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_vertices_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SpinBox_vert_size">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_size_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>590</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>620</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>141</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Central</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Parallel</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>66</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_bg_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_trans_5">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>565</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Composition</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>690</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>690</y>
      <width>430</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="MyOpenGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>841</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_comp_scale">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_turn_x">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>590</y>
      <width>50</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_screen_type">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>690</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jpeg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bmp</string>
     </property>
    </item>
   </widget>
   <zorder>frame_6</zorder>
   <zorder>frame_5</zorder>
   <zorder>frame_4</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>pushButton_File</zorder>
   <zorder>pushButton_Screenshot</zorder>
   <zorder>pushButton_save_gif</zorder>
   <zorder>label_num_vert</zorder>
   <zorder>label_num_faces</zorder>
   <zorder>label_trans</zorder>
   <zorder>label_x</zorder>
   <zorder>label_y</zorder>
   <zorder>label_y_2</zorder>
   <zorder>label_y_3</zorder>
   <zorder>label_turn</zorder>
   <zorder>label_y_4</zorder>
   <zorder>label_x_2</zorder>
   <zorder>label_edges</zorder>
   <zorder>label_size</zorder>
   <zorder>label_type</zorder>
   <zorder>label_vertices</zorder>
   <zorder>label_size_2</zorder>
   <zorder>label_type_2</zorder>
   <zorder>label_trans_5</zorder>
   <zorder>label_path</zorder>
   <zorder>lineEdit</zorder>
   <zorder>widget</zorder>
   <zorder>doubleSpinBox_comp_scale</zorder>
   <zorder>doubleSpinBox_turn_x</zorder>
   <zorder>comboBox_screen_type</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
